src/elements/netspeak-search.tsx

```diff
@@ -145,13 +152,21 @@ export class NetspeakSearch extends React.PureComponent<Props, State> {
 		} else {
 			this.setState({ loadingState: LoadingState.LOADING });
 
-			const promise = this.cancelable(
-				Netspeak.instance.search({
-					query: normalizedQuery,
-					corpus: this.props.corpusKey,
-					topk: this.props.pageSize || DEFAULT_PAGE_SIZE,
-				})
-			);
+			const promise = this.props.showExperimental
+				? this.cancelable(
+						Netspeak.neuralInstance.search({
+							query: normalizedQuery,
+							corpus: this.props.corpusKey,
+							topk: this.props.pageSize || DEFAULT_PAGE_SIZE,
+						})
+				  )
+				: this.cancelable(
+						Netspeak.instance.search({
+							query: normalizedQuery,
+							corpus: this.props.corpusKey,
+							topk: this.props.pageSize || DEFAULT_PAGE_SIZE,
+						})
+				  );
```

Comment:
I was just cleaning up the old remarks because peter quit. But it looks like there are some deeper problems here and I need to invest some time. Just to save you some time, I'll close the PR for now?

Reply:
That's fine. Just make a new one once you're done and tag me as a reviewer. Thank you!